### DISCUSSION TITLE
增加ERP系统接口以实现语音查询和填写单据

### DISCUSSION THREAD

**Discussion by HelloXMFJ** [langgenius/dify#4095] [General]
我想提出一个特性，以解决我遇到的挑战：将ERP系统的数据与dify平台进行交互。

这个请求是否与你正在面临的挑战有关？

由于企业的大部分数据都在ERP系统中，如果dify平台不能与ERP系统进行交互，就无法充分利用这些数据。

你希望看到的特性是什么？

我希望能够通过语音直接查询ERP的报表或填写单据。这个特性将使用户能够避免复杂的ERP操作，从而提高工作效率和用户体验。

这个特性将如何改善善你的工作流程/体验？

这个特性将使用户能够避免复杂的ERP操作，从而提高工作效率和用户体验。

有其他的上下文或评论吗？

无。

你能帮助实现这个特性吗？

是的，我是国内外主流ERP的实施、售前技术专家。我可以深度参与开发。

**Comment by crazywoola** [comment]
Close due to #1542
